modules/mobile/docs/openapi.yaml

```diff
@@ -2314,6 +2380,13 @@ paths:
           required: true
           schema:
             type: string
```

Comment:
not directly related to your ticket but should this letter type be an enum like the format?

Reply:
Yeah, it should be. Good catch. I'll change that.

Reply:
Done.